ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/transactions/PendingTransactions.java

```diff
@@ -275,7 +275,8 @@ private boolean shouldReplace(
     return newTransaction
             .getTransaction()
             .getGasPrice()
-            .compareTo(existingTransaction.getTransaction().getGasPrice())
+            .toUnsignedBigInteger()
```

Comment:
This one is in a performance hot zone, so we should be sure to remove the big integer boxings when we move Wei back to a delegating type.

Reply:
I brought this back now, thanks.